genome_sampler/plugin_setup.py

```diff
@@ -360,6 +361,9 @@ def _4(fmt: GISAIDDNAFASTAFormat) -> DNASequencesDirectoryFormat:
                      ' entry.',
         'metadata': 'The metadata to embed in the header.',
         'columns': 'The columns in the metadata to be used.',
+        'missing_value': "A dummy value to be used as a placeholder if any "
```

Comment:
This may be more clear, I'm not sure... I leave this one up to you. 

"Value to use to indicate missing metadata column values for sequences."